tests/functional/base_pages/question.page.js

```diff
@@ -31,11 +31,11 @@ class QuestionPage extends BasePage {
   }
 
   errorHeader() {
-    return "#main-content > div.panel.panel--error.u-mb-s > div.panel__header > div";
+    return '[data-qa="error-header"]';
   }
 
   errorNumber(number = 1) {
-    return `[data-qa="error-body"] ul > li:nth-child(${number}) > a`;
+    return `[data-qa="list-item-${number}"]`;
```

Comment:
I don't understand why this is being changed, the original is based off a data-qa selector?

Reply:
Ive already changed this back to a data-qa attribute